src/main/java/io/confluent/connect/jdbc/source/JdbcSourceTask.java

```diff
@@ -243,14 +243,21 @@ public void stop() throws ConnectException {
 
         log.debug("Returning {} records for {}", results.size(), querier.toString());
         return results;
-      } catch (SQLException e) {
-        log.error("Failed to run query for table {}: {}", querier.toString(), e);
+      } catch (SQLException sqle) {
```

Comment:
nit: As I mentioned recently on another PR, we don't tend to name exception object with acronyms in the connectors. That's a stylistic comment to let you know what's the current trend. Up to you.

Reply:
Ok, thanks!  It looks like this would need to change across the board in this connector, as `sqle` is used pervasively.  I'll leave it like this for now.